src/components/LeverageSlider/LeverageSlider.tsx

```diff
@@ -42,12 +42,13 @@ export function LeverageSlider(p: Props) {
     return { marksLabel, keyValueMap, valueKeyMap };
   }, [finalMarks]);
 
-  const sliderValue = valueKeyMap[value ?? 0] ?? DEFAULT_LEVERAGE_KEY;
+  const sliderValue = valueKeyMap[value ?? 0] ?? lerpValue2SliderKey(keyValueMap, value ?? 0);
```

Comment:
not sure about `?? 0` should not it be `x2`? I think it's not the very first key

Reply:
Agree, made `valueKeyMap[finalMarks[0]]`